docs/wiki/research/scaling/core-changes.md

```diff
@@ -0,0 +1,47 @@
+# Changes to Ethereum core to accomplish scalability
+
+## Introduction
+
+L2 Rollups need to enable the permissionless reconstruction of the L2 state. To achieve this, they must ensure Data Availability, which refers to the global ordering of inputs for transactions that have altered the state.
```

Comment:
in the intro to this section, I would differentiate between scaling of L1 itself and then achieving scalability with rollups. You can reference the original post about rollup centric roadmap and also this recent write up https://notes.ethereum.org/@mikeneuder/rcr2vmsvftv

Reply:
hey @taxmeifyoucan I added the scaling L1 section overview as requested at [commit](https://github.com/eth-protocol-fellows/protocol-studies/pull/51/commits/b215fc0e7e989635b4f174b3380a04c91f2a9a91). Is that ok now?